netbout-web/src/main/java/com/netbout/cached/CdMessages.java

```diff
@@ -146,6 +146,7 @@ public long unread() throws IOException {
         }
         /**
          * I've seen them all.
+         * @checkstyle NonStaticMethodCheck (5 lines)
          */
         @Cacheable.FlushBefore
         public void touch() {
```

Comment:
@mkordas It is the Flag.touch() method. And it has an annotation Cacheable.FlushBefore. From the annotations javadoc "Identifies a method that should flush all cached entities of **this** class/object, before being executed." I think is should no be static but an instance method.